lib/commands/performance.js

```diff
@@ -0,0 +1,339 @@
+import { errors } from 'appium-base-driver';
+import { sleep } from 'asyncbox';
+import _ from 'lodash';
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.postDismissAlert = async function () {
+  throw new errors.NotYetImplementedError();
+};
+
+const SUPPORTED_PERFORMANCE_DATA_TYPES = {
+  cpuinfo : 'the amount of cpu by user and kernel process - cpu information for applications on real devices and simulators',
```

Comment:
according to our conventions, there's no space between a key and the colon, so this should read:

```js
  cpuinfo: 'the amount...',
```